objectivec/GPBDescriptor_PackagePrivate.h

```diff
@@ -80,7 +80,7 @@ typedef struct GPBMessageFieldDescription {
   // Name of ivar.
   const char *name;
   union {
-    const char *className;  // Name for message class.
+    Class clazz;  // Class of the message.
```

Comment:
Actually, this is a breaking change.

If someone has sources generated and then updates to a runtime with this change, things won't work.  so you likely need to keep the old union member there, add this one, and then figure out a way to set a flag to the startup flows to let the runtime know which union item to expect and react accordingly.

Reply:
Fixed, and a test was added.

Reply:
Fixed, and added a test.